ESP32LapTimer/src/Wireless.cpp

```diff
@@ -0,0 +1,105 @@
+#include <WiFi.h>
+#include <Arduino.h>
+#include <ESPmDNS.h>
+#include <esp_wifi.h>
+#include <DNSServer.h>
+
+#include "TimerWebServer.h"
+#include "settings_eeprom.h"
+#include "targets/target.h"
+
+static const byte DNS_PORT = 53;
```

Comment:
I know you copied it, but ports are always uint16_t ;)

Reply:
Fixed in 6a31ca1